### ISSUE TITLE
feat: ✨ add sd-text-image

### ISSUE THREAD

**Issue by karlbaumhauer** [solid-design-system/solid#278]
### Components detailed requirements
- [ ] Props, CSS-Properties, Parts & Slots
  - [ ] ...
- [ ] Stories
  - [ ] ...

## DoR
- [ ] Item has business value
- [ ] Item has been estimated by the team
- [ ] Item is clear and well-defined
- [ ] Item is achievable within a sprint
- [ ] Item is measurable and testable upon completion (Acceptance Criteria)
- [ ] Item dependencies have been identified

## DoD
- [ ] Changes have been pushed and reviewed
- [ ] Documentation has been created/updated
- [ ] Migration Guide has been created/updated
- [ ] Implementation works successfully on `feature` branch
- [ ] Automatic tests have passed on `feature` branch
- [ ] Accessibility has been approved on `feature` branch
- [ ] Changes have been merged (`feature` branch has been deleted)
- [ ] Automatic tests have passed on `main` branch
- [ ] Accessibility has been approved on `main` branch

**Comment by mariohamann** [comment]
Feels like only CMS-related. Added `triage` label.

**Comment by coraliefeil** [comment]
Agree with @mariohamann ... no solid comp ... belongs to UI-CMS

**Comment by coraliefeil** [comment]
@EllaELSA do we have that comp in ui-cms already?

**Comment by yoezlem** [comment]
@EllaELSA @coraliefeil That would be perspectively the sd-richtext, right?

**Comment by yoezlem** [comment]
@karlbaumhauer will talk with Brand and CMS Teams.

**Comment by karlbaumhauer** [comment]
We discussed the topic in a bigger round with Brand and CMS POs.

We shared the challenge with the floating text feature as this is actually a static design element and should not be used in responsive web design. The feature will not be doable in a acceptable time-box and will therefore most probably not be continued in the SDS.
To finally take this decision, we are waiting for some feedback from the authors on how they actually use the component. Technical evaluations are not really possible as 1st, the text floating is enabled by default and therewith cannot be uses as filter and 2nd, the component is mostly used in a wrong context due to the missing component-user-guide in the CMS.

Further decisions will be taken after we got some feedback from the authors.
